tools.go

```diff
@@ -0,0 +1,22 @@
+package vpphelper
+
+import (
+	"strings"
+	"text/template"
+)
+
+type VPPConfigParameters struct {
+	DataSize int
+	RootDir  string
+}
+
+func NewVPPConfigFile(confTemplate string, params VPPConfigParameters) string {
```

Comment:
Note confTemplate is already const. See at `vppConfContents`
```suggestion
func NewVPPConfigFileSource(params VPPConfigParameters) string {
```

Reply:
I think we should use the `confTemplate` parameter, because we can set any other config template using `WithVppConfig()` option